python/requirements.txt

```diff
@@ -14,6 +14,5 @@
 # limitations under the License.
 #
 flask==0.10.1
-pymongo==2.6.3
 apscheduler==2.1.2
-couchdb==0.9
\ No newline at end of file
+couchdb==1.1
```

Comment:
I would keep the current library and merge the changes. We should open an issue to adapt to the Cloudant library